.github/workflows/continuous-integration.yml

```diff
@@ -126,7 +125,6 @@ jobs:
     strategy:
       matrix:
         php-version:
-          - "7.1"
```

Comment:
If PHP 7.2 is still supported, should it be used for testing fixes?

Reply:
It is, no? This removes 7.1 testing

Reply:
I agree that we should not just remove the 7.1 job, but replace it with a 7.2 one instead. Since PHP 7.2 is going to be the lowest supported version, it should be represented in this matrix.

Reply:
I missed this one, good catch!